android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/AuthenticationManager.java

```diff
@@ -269,35 +251,26 @@ public UserSessionInfo getUserSessionInfo() {
         // TODO: a way to distinguish if session is null because we haven't signed on, or if it
         // was invalidated
         UserSessionInfo session = apiClientProvider.getUserSessionInfoProvider()
```

Comment:
I don't think it is a good idea to have the UserSession exists in multiple places.  It leads to confusion about when you should use one or the other.  Is there a way we could just store it in the accountDao?